packages/paste-core/components/card/stories/index.stories.tsx

```diff
@@ -1,12 +1,40 @@
 import * as React from 'react';
 import {storiesOf} from '@storybook/react';
 import {Heading} from '@twilio-paste/heading';
+import {withKnobs, select} from '@storybook/addon-knobs';
 import {Text} from '@twilio-paste/text';
 import {Card, CardFooter} from '../src';
 
+import {Padding, Space} from '@twilio-paste/types';
+
+const space = [
```

Comment:
You can replace this with 
```js
import {DefaultTheme, ThemeShape} from '@twilio-paste/theme-tokens';

const spaceOptions = Object.keys(DefaultTheme.space);

....

const paddingValue = select('padding', spaceOptions, 'space10') as keyof ThemeShape['space'];

... 

<Card padding={paddingValue}>
```

so that this list doesn't go stale and we maintain a single source of truth.  (Box' story does this)

Reply:
`'@twilio-paste/theme-tokens'` doesn't need to go into the package.json because Storybook has access to it.